dataland-frontend/src/assets/content.json

```
@@ -26,39 +26,39 @@
           "cards": [
             {
               "icon": "BzUkAtdcKvg",
-              "title": "Sven Schubert,",
+              "title": "Sven Schubert",
               "text": " CEO of Envoria",
               "date": "\"Dataland is the only platform we know that is open to everyone and based on a non-profit business model.\""
             },
-            {
-              "icon": "NqEjkz-3hPw",
-              "title": "Jonas Weisbach,",
-              "text": "Business Development Manager ESG\n at Metzler Asset Management",
-              "date": "\"We believe that open source, reliable and comparable data is the single best way to identify the companies that walk the walk.\""
-            },
             {
               "icon": "rCkISSQreCI",
-              "title": "Dr. Egbert Schark,",
+              "title": "Dr. Egbert Schark",
               "text": "Founder and Managing Director\n at d-fine GmbH",
               "date": "\"I believe the fascinating idea is worth supporting. Join the mission! Join Dataland!\""
             },
             {
               "icon": "g40FkVQZjUc",
-              "title": "Daniel Sailer,",
+              "title": "Daniel Sailer",
               "text": " Head of Sustainable Investment Office\n at Metzler Asset Management GmbH",
               "date": "\"The Pathways to Paris PoC can be a simple way to make sure Dataland becomes the data platform you need.\""
             },
             {
               "icon": "Jx1v2Z-9cBw",
               "title": "Dr. Anna-Lisa Schwarz",
-              "text": "Managing Director at Werte-Stiftung ",
+              "text": "Managing Director at Werte-Stiftung",
               "date": "\"Dataland will help solve data issues by ensuring transparent, open and fair access to sustainability data.\""
             },
             {
               "icon": "WJg2KKUVch4",
               "title": "Rudolf Siebel",
-              "text": "Managing Director at BVI German Fund Association ",
-              "date": "\"[Through Dataland], we hope that data availability, coverage and quality is improved, for the benefit of the users, the corporations and society overall.\""
+              "text": "Managing Director at BVI German Fund Association",
+              "date": "\"Through Dataland, we hope that data availability, coverage and quality is improved, for the benefit of the users, the corporations and society overall.\""
```

```suggestion
              "date": "\"[Through Dataland], we hope that data availability, coverage and quality is improved, for the benefit of the users, the corporations and society overall.\""
```

Since we don't want to misquote Rudi it is important that we keep the [...] as we are slightly morphing the meaning of what he said with our quote.

---

Okay. I thought that the brackets were sort of an inconsistency/legacy (like e.g. previously "Dataland" was put in brackets to highlight it) and could be removed. Watching the video, I understand that this is sort of an "editorial change" to render the quote meaningful without the original context.

Nevertheless, it still is a bit inconsistent, i.e. we do not highlight consequently where single words were added or deleted for editorial purposes. But I guess that's okay.